cpp/src/io/parquet/predicate_pushdown.cpp

```diff
@@ -278,7 +277,6 @@ class stats_expression_converter : public ast::detail::expression_transformer {
                  "Statistics AST supports only left table");
     CUDF_EXPECTS(expr.get_column_index() < _num_columns,
                  "Column index cannot be more than number of columns in the table");
-    _stats_expr = std::reference_wrapper<ast::expression const>(expr);
```

Comment:
These don't need to be pushed to the `ast::tree` since we will be filtering with the new columns in the StatsAST table.